.github/workflows/ci.yml

```diff
@@ -14,19 +14,19 @@ jobs:
     runs-on: windows-2019
     steps:
       - name: Checkout
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v3.0.2
         with:
           fetch-depth: 0
       - name: Setup .NET SDK
-        uses: actions/setup-dotnet@v1.7.2
+        uses: actions/setup-dotnet@v2.1.0
         with:
           dotnet-version: 3.1.x
       - name: Add msbuild to PATH
```

Comment:
This seems different from other repos. Any idea why it cannot use `dotnet` to build?